wordle-web/components/PossibleWords.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div>
+    <v-select
+      clearable
+      :label="labelText"
+      v-model="selectedWord"
+      :items="wordList"
+      variant="solo-filled"
+      :menu-props="{ bottom: true }"
+    ></v-select>
+  </div>
+</template>
+
+
+<script setup lang="ts">
+import { WordService } from "~/scripts/wordService";
+import { ref, onMounted, watch, inject, computed } from "vue";
+import { Letter, LetterState } from "~/scripts/letter";
+import { Word } from "~/scripts/word";
+import { Game } from "~/scripts/game";
+
+const labelText = "Possible Words: 10000";
+
+const game = inject("GAME") as Game;
+
+const { emit } = defineEmits(["selectedWord"]);
+
+const wordList = ref<string[]>([]);
+let selectedWord = ref<string | null>(null);
+
+watch(() => game?.guesses, (newGuesses) => {
+  if (newGuesses){
+    const stateArray = getStateArray(newGuesses);
+    wordList.value = WordService.validGuessedWords(newGuesses, stateArray);
+  }
```

Comment:
```suggestion
  () => game.guesses,
```

I don't see a scenario where game could be `null` or `undefined` if we are explicitly providing it so you can remove it imo.

Reply:
I completely refactored the way I made this so that should not be a issue anymore. I did this when i was running into a bunch of problems.